src/platform_impl/windows/window.rs

```diff
@@ -698,6 +698,14 @@ impl Window {
         Some(self.window_state_lock().current_theme)
     }
 
+    #[inline]
+    pub fn title(&self) -> String {
+        let len = unsafe { GetWindowTextLengthW(self.window.0) };
+        let mut buf = vec![0; (len + 1) as usize];
+        unsafe { GetWindowTextW(self.window.0, buf.as_mut_ptr(), len) };
+        String::from_utf16_lossy(&buf[..(len - 1) as _])
```

Comment:
Isn't what you're doing is basically `OsString` here? Though, you might be fine with raw buffers.

Reply:
do you want to the API to return `OsString` instead or you want me to go through `OsString` to `String`

Reply:
OsString to String.

Reply:
Wouldn't that result in extra allocation?

Reply:
I guess it's fine then.